postgis-17.yaml

```diff
@@ -1,5 +1,5 @@
 package:
-  name: postgis
+  name: postgis-17
   version: 3.5.0
   epoch: 1
```

Comment:
when we change the package name here should we reset the epoch?

Reply:
totally, thanks for catching!